### DISCUSSION TITLE
docker for latest version/s?

### DISCUSSION THREAD

**Discussion by mdsumner** [zarr-developers/VirtualiZarr#293] [Q&A]
I need some help with getting latest deps installed, I've been using pangeo docker successfully but having trouble getting latest versions installed, it's a bit indirect because I'm running jupyterlab via docker on an HPC . 

If someone's maintaining a docker image for linux that has latest VirtualiZarr for linux that would be really helpful. Thanks!

**Comment by maxrjones** [comment]
It's a bit bloated at ~2.7GB, but I've been working off https://quay.io/repository/developmentseed/veda-optimized-data-delivery-image (source code at https://github.com/developmentseed/veda-optimized-data-delivery-image). I use this on the [VEDA Hub](https://docs.openveda.cloud/nasa-veda-platform/getting-access.html).

I also might make a lighter weight version with minimal deps for virtualizing NetCDF4 to accompany https://github.com/developmentseed/virtualize-nex-gddp-cmip6.
